markdown.ipynb

```diff
@@ -4,21 +4,22 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `markdown.ipynb` Cell 33 Line 1_
```markdown
This is a regular paragraph followed by some inline HTML
```
A regular pragraph single review comment

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/24)._

Reply:
> a regular paragraph single review comment

This is a response

Reply:
response

Reply:
my reply in github

Reply:
oh I see it now